dotnet/src/SemanticKernel.UnitTests/Planning/PlanSerializationTests.cs

```diff
@@ -15,7 +15,7 @@ namespace SemanticKernel.UnitTests.Planning;
 
 public sealed class PlanSerializationTests
 {
-    private readonly Mock<IKernel> _kernel = new();
+    private Mock<IKernelExecutionContext> _kernelContext = new();
```

Comment:
Why the `readonly` removed, seemed nowhere updates it

Reply:
Added.